go.mod

```diff
@@ -1,25 +1,85 @@
 module k8s.io/org
 
-go 1.13
+go 1.20
 
 require (
+	github.com/bmatcuk/doublestar v1.3.4
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-git/go-git/v5 v5.4.2
-	github.com/sirupsen/logrus v1.4.2
-	github.com/spf13/cobra v0.0.5
-	k8s.io/apimachinery v0.0.0-20190817020851-f2f3a405f61d
-	k8s.io/test-infra v0.0.0-20191222193732-de81526abe72
-	sigs.k8s.io/yaml v1.1.0
+	github.com/go-git/go-git/v5 v5.6.1
+	github.com/sirupsen/logrus v1.9.0
+	github.com/spf13/cobra v1.7.0
+	k8s.io/apimachinery v0.27.4
+	k8s.io/test-infra v0.0.0-20230726003218-c95b43963de2
+	sigs.k8s.io/yaml v1.3.0
 )
 
-// Pin all k8s.io staging repositories to kubernetes-1.15.3.
+// Pin all k8s.io staging repositories to kubernetes v1.27.4
 // When bumping Kubernetes dependencies, you should update each of these lines
-// to point to the same kubernetes-1.x.y release branch before running update-deps.sh.
+// to point to the same kubernetes v0.KubernetesMinor.KubernetesPatch version
+// before running update-deps.sh.
```

Comment:
@MadhavJivrajani, @palnabarun I was thinking of removing above lines of comment from the `go.mod` file, since `update-deps.sh` hack file is no longer present in the repo (was removed as part of https://github.com/kubernetes/org/pull/1278).

WDYT? Is there something else that is doing the job of  `update-deps.sh` script currently and can be replaced with, in the comments?

Reply:
SGTM!

Reply:
Thanks, pushed changes in the latest commits refresh. :+1: